lib/router.js

```diff
@@ -129,25 +121,22 @@ Router.prototype._loadModule = function(modDef, globals) {
         } else if (/\.yaml$/.test(modDef.path)) {
             modType = 'spec';
         }
+    } else if (modDef.spec && !modType) {
+        modType = 'inline';
     }
 
     // Determine the module's load path
     switch (modType) {
         case 'file':
         case 'spec':
-            if (modDef.path && /^\//.test(modDef.path)) {
-                // Absolute path
-                loadPath = modDef.path;
-            } else if (modDef.path) {
-                // Relative path or missing
-                loadPath = this._options.appBasePath + '/' + modDef.path;
-            } else {
-                throw new Error('Unknown module path');
-            }
+            loadPath = modDef.path;
```

Comment:
Ups, I went too far with the refactoring.. It's not required any more only for the `file` module type. Will revert.